nemo/collections/asr/parts/submodules/subsampling.py

```diff
@@ -264,7 +265,19 @@ def forward(self, x, lengths):
         )
         x = x.unsqueeze(1)
 
-        x = self.conv(x)
+        # avoiding a bug / feature limiting indexing of tensors to 2**31
```

Comment:
an idea: link to https://github.com/pytorch/pytorch/issues/80020 directly here - would be easier to understand the motivation from just reading the code

Reply:
good idea, thank you, and done.